src/screens/Home/Home.tsx

```diff
@@ -167,7 +166,7 @@ function Home(props: HomeProps) {
                 <Button label='Current Period' startIcon={<DescriptionIcon />} />
               </Link>
             </div>
-            <Link to={'/financial-summary'}>
+            <Link to={'/financial-summaries'}>
```

Comment:
Both The way we pass `report` and the fact that we can't pass stuff through a `button` both don't really translate well to going immediately to the current financial summary.


I'm down to merge and figure this out in a subsequent PR since I think what we have now is good :^)